kura/distrib/config/kura.build.properties

```diff
@@ -89,19 +89,19 @@ org.eclipse.kura.example.wire.math.multiport.provider.version=1.3.0
 org.eclipse.kura.example.tamper.detection.version=1.3.0
 org.eclipse.kura.web2.version=2.3.0
 org.eclipse.kura.wire.camel.version=1.3.0
-org.eclipse.kura.wire.component.provider.version=2.0.1-SNAPSHOT
+org.eclipse.kura.wire.component.provider.version=2.0.1
 org.eclipse.kura.wire.h2db.component.provider.version=2.3.0
 org.eclipse.kura.wire.db.component.provider.version=1.1.0
 org.eclipse.kura.wire.script.filter.provider.version=1.3.0
 org.eclipse.kura.wire.helper.provider.version=1.3.0
 org.eclipse.kura.wire.provider.version=1.3.0
 org.eclipse.kura.rest.provider.version=1.4.0
-org.eclipse.kura.rest.asset.provider.version=1.3.1-SNAPSHOT
+org.eclipse.kura.rest.asset.provider.version=1.3.1
```

Comment:
I think we have upticked the wrong bundle, `org.eclipse.kura.asset.provider` should be upticked instead of this one.

Reply:
You are right. Should be fixed now